tests/conftest.py

```diff
@@ -169,7 +169,7 @@ def pg_params(pg_server):
     return dict(**pg_server['pg_params'])
 
 
-@pytest.yield_fixture()
+@pytest.fixture()
```

Comment:
Can you please review `()` here and in other similar places, better to be consistent everywhere.

Reply:
done